indico/modules/announcement/templates/display.html

```diff
@@ -2,15 +2,17 @@
     {% if type == 'html' %}
         {{ message | safe }}
     {% else %}
-        <div class="announcement-bar weak-hidden {{ type }}" data-hash="{{ message | crc32 }}">
```

Comment:
Removing the initially hidden styling here results in a short flash of the announcement bar on every page load after you dismissed it, which is quite annoying.